crates/re_types/definitions/rerun/blueprint/archetypes/container_blueprint.fbs

```diff
@@ -10,13 +10,16 @@ namespace rerun.blueprint.archetypes;
 /// The top-level description of the Viewport.
 table ContainerBlueprint (
     "attr.rerun.scope": "blueprint",
-    "attr.rust.derive": "Default",
```

Comment:
The old code had a bug, which defaulted the `container_kind` to the invalid value `0`